lib/cmd/common/configuration.js

```diff
@@ -27,22 +27,22 @@ function configuration(argv, cb) {
 
   // defaults.. 'list all' operation if 1 args passed, 'list <dest>' if 2..
   if (args.length === 1) {
-    return list(fhc.appId(args[0]), 'all', cb);
+    return list(args[0], 'all', cb);
```

Comment:
The idea of this PR is to remove the legacy code into the V2. However, the commands into common will be migrated for the V3 standard and in order to use yargs properly. But it will be done per command. For this command, for example, we already have the PR: https://github.com/feedhenry/fh-fhc/pull/375